README.md

```diff
@@ -0,0 +1,33 @@
+# Analysis Manual for pytpc
+
+This documentation manual provides steps for installation and use of the [pytpc package](https://github.com/ATTPC/pytpc) that provides Python modules for reading, analyzing, and fitting data from the Active-Target Time Projection Chamber (AT-TPC) at the [NSCL] (http://nscl.msu.edu>). The documentation is hosted online by ReadTheDocs [ReadTheDocs](https://github.com/ATTPC/analysis-docs) but steps for offline compilation are listed below.
```

Comment:
The ReadTheDocs URL should be https://attpc-analysis.readthedocs.io/en/latest/.

Also, can you add a badge for the RTD status at the top somewhere? The Markdown code for the badge is

`[![Documentation Status](https://readthedocs.org/projects/attpc-analysis/badge/?version=latest)](http://attpc-analysis.readthedocs.io/en/latest/?badge=latest)`

which renders the badge like this:

[![Documentation Status](https://readthedocs.org/projects/attpc-analysis/badge/?version=latest)](http://attpc-analysis.readthedocs.io/en/latest/?badge=latest)